### PR TITLE
Allow testing with any number of concurrent ginkgo processes

### PR DESCRIPTION
**Pull request by akalenyu**
### What this PR does
This is the only tech debt that remains in order for anyone
to be able to run kubevirt tests with any amount of concurrent ginkgo processes
(`export KUBEVIRT_E2E_PARALLEL_NODES=<NUM>` on the host executing the tests).

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

**Review by dhiller — COMMENTED** [review]
/approve

Thanks @akalenyu for this great cleanup!

**Comment by akalenyu** [comment]
> /approve
> 
> Thanks @akalenyu for this great cleanup!

Thanks, for some reason this didn't get approved though

File: tools/manifest-templator/manifest-templator.go [hunk 1]
```diff
@@ -120,6 +121,7 @@ func main() {
 	runbookURLTemplate := flag.String("runbook-url-template", "", "")
 	featureGates := flag.String("feature-gates", "", "")
 	infraReplicas := flag.Uint("infra-replicas", 0, "")
+	testImageReplicas := flag.String("test-image-replicas", "0", "")
```
Comment (0xFelix) [inline tools/manifest-templator/manifest-templator.go:124]: nit: Should this default to `6`?

**Review by 0xFelix — COMMENTED** [review]
@dhiller is no longer an approver IIRC? :(

File: tools/manifest-templator/manifest-templator.go (hunk 1 above)
Comment (akalenyu) [reply tools/manifest-templator/manifest-templator.go:124]: I don't think so, 6 is like a CI default only for kubevirt/kubevirt

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Review by brianmcarey — COMMENTED** [review]
/lgtm

thanks @akalenyu

**Comment by akalenyu** [comment]
> @dhiller is no longer an approver IIRC? :(

yeah I am pretty sure Daniel can approve these?

**Comment by 0xFelix** [comment]
https://github.com/kubevirt/kubevirt/pull/11521

I don't think so anymore

**Comment by akalenyu** [comment]
> #11521
> 
> I don't think so anymore

But this PR is sig-buildsystem related

**Comment by 0xFelix** [comment]
`/images`, `/manifests` and `/tools` are not covered by `sig-buildsystem`. Only the `/hack` directory is.

**Comment by xpivarc** [comment]
> /approve
> 
> Thanks @akalenyu for this great cleanup!

/approve

**Pull request merged**